dev-docs/workflows/build-test-run.md

```diff
@@ -28,14 +28,21 @@ mkdir build
 cd build
 # build required binaries for a dev build
 # and symlink them into the current directory
-bazel run //:devbuild
+# also push the built container images
+bazel run //:devbuild --container_prefix=ghcr.io/USERNAME
```

Comment:
Maybe we should add a warning here that users have to set the images to public on the first push. This may not be obvious.